cmake/CMakeResourceDependencies.cpp

```diff
@@ -20,14 +20,12 @@
 
```

Comment:
was this file accidentally brought in to the PR? This change doesn't appear to be relevant to manifest parsing.

Reply:
I believe this came in as part of a merge of 349_Fix_Makefile_builds_on_macos.

Reply:
this file should be removed from the PR then.

Reply:
Hopefully it's now removed. I restored the state of the file to be as it was before the merge.